hail.md

```diff
@@ -414,6 +491,40 @@ This will print a link to the [CI dashboard](https://ci.hail.populationgenomics.
 
 **Warning**: Some changes that involve a database migration will result in the batch service being shut down. You'll then need to [bring it back up manually](https://github.com/hail-is/hail/blob/main/dev-docs/development-process.md#merge--deploy).
 
+### Hail Batch SQL database
+
+Sometimes there is a need to view SQL content of Hail Batch database. To be able to do that, add the following function to your local .zshrc file:
+
+```bash
+kube-hail() {
+# Find all pods that match the input name, get the first one
+local pod_name=$(kubectl get pods --no-headers=true | grep "$1" | head -n 1 | awk '{print $1}')
+
+# Check if a pod name was found
+if [ -z "$pod_name" ]; then
+echo "No matching pod found"
```

Comment:
Looks like some indentation fell off here (and off the three lines under `else`), but no biggie.

Reply:
It would work as is, it is bash code, but good point. Had updated it.